pkg/approver/approver.go

```diff
@@ -45,4 +45,8 @@ type Interface interface {
 	// Webhook implements admission functions for CertificateRequestPolicy
 	// resources.
 	Webhook
+
+	// Reconciler is responsible for determining the status of a
```

Comment:
Yes, the policy controller will aggregate the Reconciler responses. If all return Ready, then the controller updates the policy as Ready, NotReady otherwise.